src/main/app/jurisdiction/connections.ts

```diff
@@ -92,7 +93,19 @@ const hasResidualJurisdiction = (data, connections) => {
   );
 };
 
-export const addConnection = (data: Partial<CaseWithId>): JurisdictionConnections[] => {
+export const addConnections = (reqUrl: string, data: Partial<CaseWithId>): JurisdictionConnections[] => {
```

Comment:
I'm not sure I like this approach. It feels like we're cramming situation specific logic into the wrong place. My gut feel is that the calling method should know whether or not it should be behave this way. I don't have a full understanding but I would suggest the calling code that needs both variations of this method should be something like `addConnectionsBasedOnQuestions().concat(addConnections())`